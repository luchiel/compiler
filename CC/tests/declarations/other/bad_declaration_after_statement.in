void main()
{
int a;
a = 5;
float b;
}