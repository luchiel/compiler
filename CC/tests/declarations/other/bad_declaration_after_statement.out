Error: ParseError:6:1 Declaration after statements
